### ISSUE TITLE
CocoaPods could not find compatible versions for pod "Firebase/Core"

### ISSUE THREAD

**Issue by newuser44** [chemerisuk/cordova-plugin-firebase-analytics#88]
I'm getting this error trying to set things up. 

```
[!] CocoaPods could not find compatible versions for pod "Firebase/Core":
  In snapshot (Podfile.lock):
    Firebase/Core (= 5.4.0, ~> 5.0)

  In Podfile:
    Firebase/Core (~> 5.4.0)

Specs satisfying the `Firebase/Core (~> 5.4.0), Firebase/Core (= 5.4.0, ~> 5.0)` dependency were found, but they required a higher minimum deployment target.

```

Not sure which version needs to be fixed,  Firebase/Core or CoocaPods version?  

pod --version gives me 1.5.3

When I try to build I get the error   
FirebaseCore/FirebaseCore.h' file not found

**Comment by kylemellander** [comment]
I am also experiencing this

**Comment by BinaryBlock** [comment]
I too am having the same issue.

**Comment by yosuke1985** [comment]
I also....

**Comment by chemerisuk** [comment]
Try
```
cd platforms/ios && pod update
```

**Comment by jashanpj** [comment]
> I am also experiencing this

Did you get the solution?

**Comment by BinaryBlock** [comment]
> > I am also experiencing this
> 
> Did you get the solution?

I ended up using the [ionic native google analytics](https://ionicframework.com/docs/native/google-analytics/), due to time constraints and not finding a solution.

**Comment by MumblesNZ** [comment]
I solved this error for myself by setting my minimum deployment target to iOS 8.0.

**Comment by JustasKuizinas** [comment]
Similar problem here 
```
Unable to satisfy the following requirements:

- `GoogleUtilities/MethodSwizzler (~> 5.2.0)` required by `FirebaseAnalytics (5.1.1)`

Specs satisfying the `GoogleUtilities/MethodSwizzler (~> 5.2.0)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by MumblesNZ** [comment]
@JustasKuizinas what's your iOS minimum deployment target?

**Comment by JustasKuizinas** [comment]
@MumblesNZ 
```
 <platform name="ios">
        <preference name="deployment-target" value="10.0"/>
```

**Comment by hanishassim** [comment]
I am also having this issue.. anyone here managed to resolved it?

**Comment by VrushaliWani** [comment]
Try deleting _Podfile.lock_ and then `pod update`

**Comment by alahji7** [comment]
same issue, none of the above suggestions worked..

**Comment by matthijsbierman** [comment]
Known issue: https://github.com/firebase/firebase-ios-sdk/issues/1845
Fixed in Firebase 5.8.1, but this plugin appears to use **5.6**:
`<framework src="Firebase/Core" type="podspec" spec="~> 5.6"/>`

**Comment by askona** [comment]
I had this issue because of pod versions of firebase/core and others in plugin.xml cordova-plugin-firebasex has different versions. Solved by manually changing version to highest of all plugins, using Firebase pods

**Comment by mfsaglam** [comment]
Just update the pods, and retry installing. Worked for me.

**Comment by glennweb** [comment]
I needed to delete `Podfile.lock` and run `pod install` again.

**Comment by dom98111** [comment]
> I am unable to find Podfile.lock. Can anyone please guide me to the location of it, i can only see Podfile not Podfile.lock
> 
> I am getting this error
> 
> ```
> [!] CocoaPods could not find compatible versions for pod "Firebase/Analytics":
>   In Podfile:
>     CordovaPluginsStatic (from `../capacitor-cordova-ios-plugins`) was resolved to 2.4.8, which depends on
>       Firebase/Analytics (~> 8.8.0)
> 
>     Firebase/Analytics
> 
> Specs satisfying the `Firebase/Analytics, Firebase/Analytics (~> 8.8.0)` dependency were found, but they required a higher minimum deployment target.
> ```

Hi! I have the same problem. Did you manage to solve the problem?

**Comment by phoenix503** [comment]
No i switched to [Capacitor Plugin](https://github.com/capacitor-community/firebase-analytics)
